### PR TITLE
feature(zjow): add trackback log for subprocess env manager

### PR DESCRIPTION
**Pull request by zjowowen**
#532

### CODE REVIEW

File: ding/envs/env_manager/subprocess_env_manager.py [hunk 1]
```diff
@@ -398,6 +398,7 @@ def reset(self, reset_param: Optional[Dict] = None) -> None:
                     self._env_seed[env_id] = None  # seed only use once
                 except BaseException as e:
                     logging.warning("subprocess reset set seed failed, ignore and continue...")
+                    logging.warning("subprocess exception traceback: \n" + traceback.format_exc())
```
Comment (PaParaZz1) [inline ding/envs/env_manager/subprocess_env_manager.py:401]: merge two logs

File: ding/envs/env_manager/subprocess_env_manager.py [hunk 2]
```diff
@@ -439,6 +440,7 @@ def reset_fn():
                 reset_fn()
                 return
             except BaseException as e:
+                logging.warning("subprocess exception traceback: \n" + traceback.format_exc())
```
Comment (PaParaZz1) [inline ding/envs/env_manager/subprocess_env_manager.py:443]: don't use warning, rather than info, they are catched exception

**Review by PaParaZz1 — APPROVED** [review]

File: ding/envs/env_manager/subprocess_env_manager.py (hunk 1 above)
Comment (zjowowen) [reply ding/envs/env_manager/subprocess_env_manager.py]: fixed.

File: ding/envs/env_manager/subprocess_env_manager.py (hunk 2 above)
Comment (zjowowen) [reply ding/envs/env_manager/subprocess_env_manager.py]: fixed.

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**